<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1921</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pedidos Pendientes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/logo.png</normaloff>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-60</y>
      <width>2121</width>
      <height>1171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
 background: #A3A3A3;
 color: white;
 padding: 10px;
border-radius: 1px;

 }

 QTabBar::tab:selected {
  background: #2FA9E0;
 }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Pedidos">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { 
   border: none;
}</string>
     </property>
     <attribute name="title">
      <string>Pedidos</string>
     </attribute>
     <widget class="QLabel" name="lbl_cantpedidos_2">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>990</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Quicksand</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Scroll</string>
      </property>
     </widget>
     <widget class="QLabel" name="Acqualogo">
      <property name="geometry">
       <rect>
        <x>1680</x>
        <y>980</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/acqualogo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>671</width>
        <height>931</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Quicksand</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget { outline: none; border:0px solid gray; color: black;background: #F2F2F2;;border-radius: 10px;padding:10px;}
QListWidget::Item { width: 50px; height: 50px; }
QListWidget::Item:hover { background: #2FA9E0; color: white;border-radius:0px; }
QListWidget::item:selected { background: #e7e7e7;color: #30A2FF; }
QListWidget::QScrollBar:vertical {              
            border: none;
            background:white;
            width:3px;
            margin: 0px 0px 0px 0px;
        }
</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>1221</width>
        <height>931</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="A_Grupo2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1211</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="lbl_numero">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Quicksand</family>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:#F2F2F2;
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_empresa">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>931</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Quicksand</family>
          <pointsize>36</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F2F2;
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="check_visto">
        <property name="geometry">
         <rect>
          <x>1160</x>
          <y>30</y>
          <width>41</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator {width:41px;height: 31px;}
QCheckBox::indicator:checked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxcheck.png);
}
QCheckBox::indicator:unchecked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxempty.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_visto">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>0</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Quicksand</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Visto</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="A_Grupo1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background: #F2F2F2;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="lbl_requerimiento">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Quicksand</family>
          <pointsize>22</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;</string>
        </property>
        <property name="text">
         <string>Requerimiento:</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="cantidad">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0;</string>
       </property>
       <property name="text">
        <string>Cant</string>
       </property>
      </widget>
      <widget class="QLabel" name="descripcion">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>1151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Descripcion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QListWidget" name="listdesc">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>360</y>
         <width>1121</width>
         <height>561</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F2F2F2;
border-radius: 10px;
padding: 10px;

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_entrega">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>1071</width>
         <height>81</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Chivo</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_retira">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/delivery-box.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_entrega">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/truck.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listcant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>61</width>
         <height>561</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#F2F2F2;
border-radius: 10px;
padding: 10px;</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_advertencia">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/advertencia.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_retira">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Chivo</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Retira</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_pendiente">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>140</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/pendiente.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_completo">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>140</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/verificado.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_pagopendiente">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>140</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/pago_pendien.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_pagorealizado">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>140</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/pago_check.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_pagoerror">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>140</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/pago_no_existente.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_numerofactura">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>140</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background: #F2F2F2;
padding: 8px;</string>
       </property>
       <property name="text">
        <string>1124554</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_pagorechazado">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>140</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/pago_rechazdo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_reqsin">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>140</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/sinrequerimiento.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_vistoaremitir">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>190</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Quicksand</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remitible</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_remitir">
       <property name="geometry">
        <rect>
         <x>920</x>
         <y>140</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator {width:41px;height: 31px;}
QCheckBox::indicator:checked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxcheck.png);
}
QCheckBox::indicator:unchecked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxempty.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="check_scroll">
      <property name="geometry">
       <rect>
        <x>1550</x>
        <y>990</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {width:37px;height: 27px;}
QCheckBox::indicator:checked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxcheck.png);
}
QCheckBox::indicator:unchecked
{
    image: url(//servidor/Users/Public/Acquatron Compartida/Andones/Resources/checkboxempty.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_cantpedidos">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>990</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Quicksand</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pedidos Pendientes:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_help">
      <property name="geometry">
       <rect>
        <x>1610</x>
        <y>980</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/help.png</normaloff>//servidor/Users/Public/Acquatron Compartida/Andones/Resources/help.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>500</width>
        <height>36</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="General">
     <attribute name="title">
      <string>Requerimientos</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
